build-sdist/action.yml

```
@@ -0,0 +1,23 @@
+name: 'Build source distribution'
+description: 'Build a Python source distribution using build and upload the resulting artifact'
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v2

```

Some packages use `setuptools_scm`, so you need to fetch all the tags.
```suggestion
    - uses: actions/checkout@v2
      with:
        fetch-depth: 0
```

---

Great catch, this is definitely the sort of thing I wouldn't have realized on my own!

---

Great catch, this is definitely the sort of thing I wouldn't have realized on my own!